src/examples/graph.js

```diff
@@ -464,15 +464,10 @@ class Graph extends React.Component<IGraphProps, IGraphState> {
     });
   };
 
-  onPasteSelected = () => {
-    if (!this.state.copiedNode) {
-      console.warn(
-        'No node is currently in the copy queue. Try selecting a node and copying it with Ctrl/Command-C'
-      );
-    }
-
+  // Pastes the copied node to mouse position
+  onPasteSelected = (x: Number, y: Number, node: INode) => {
```

Comment:
With the previous file change:

```
// Tuple type [number, number] should work, but if it doesn't try number[] instead
onPasteSelected = (node: INode, mousePosition: [number, number]) => {
```

Reply:
Will this implementation not collide with any of your team's internal plans for this project?

Reply:
> Will this implementation not collide with any of your team's internal plans for this project?

This implementation actually works with the future changes, because we're planning on multiple-node and edge copy/paste. If we do multi-select then it needs to follow the mouse.